ci/Jenkinsfile_docker_cache

```diff
@@ -38,6 +38,7 @@ core_logic: {
         timeout(time: total_timeout, unit: 'MINUTES') {
           utils.init_git()
           sh "python3 ./ci/docker_cache.py --docker-registry ${env.DOCKER_CACHE_REGISTRY}"
+          sh "cd ci && docker-compose -f docker/docker-compose.yml build --parallel && docker-compose -f docker/docker-compose.yml push "
```

Comment:
This seems not part of your PR, right? Please do rebase properly.

Reply:
Sorry, did a wrong rebase. Thanks for pointing out. Updated